exercise_2/Course.cpp

```diff
@@ -0,0 +1,20 @@
+//
+// Created by cdd on 16.07.19.
+//
+
+#include "Course.h"
+#include <string>
+#include <sstream>
+
+
+Course::Course(const int &id,const std::string &name,const int &teacherId):_id(id),_name{name},_teacherId{teacherId} {
+}
+int Course::getId() const {
+    return _id;
+}
+
+std::string Course::getFormatted() const {
+    std::ostringstream  formatedData;
```

Comment:
That code is being repeated all over other classes. Have you been thinking about making it common for all of the types?